---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* Stanford University • Major: Computer Science, Minors: Biology and Sustainability • GPA: 4.02/4.0 • June 2024
  * Courses: Data Structures and Algorithms, Computer Architecture, Operating Systems, Linear Algebra, Organic Chemistry, Biochemistry, Cell Biology, Computational Protein Modeling, Computational Biology, Machine Learning, Natural Language Processing
  * Organizations: Stanford TreeHacks (Sponsorships Team), Stanford Association for Computing Machinery, Stanford Students in Biodesign and Biopharma (Conference Chair), Stanford Hindu Students Council (President)
  * Honors: Pear VC Garage Fellow, Stanford Energy in Hawaii Technology Fellow

Work Experience
======
* **Stanford AI Laboratory (SAIL): Dror Lab • December 2022 - Present**
  * Improving ligand conformer generation in protein-ligand docking simulations using novel AI techniques alongside
physics-based pharmacophore modeling

* **Data Science/Machine Learning Intern at BigHat Biosciences • June 2022 - September 2022**
  * Developed decision-tree algorithm incorporating Bayesian statistical methods to improve antibody engineering,
decreasing assay technical failure rate on platform from ~50% to ~10% and improving ML modeling
  * Created grammar using decision tree to explain why antibodies failed quality control leading to cost savings
  * Built LSTM-GANs to benchmark effects of incorporating grammar into ML affinity modeling

* **Stanford Biomedical Informatics Research (BMIR): Khatri Lab • February 2022 - September 2022**
  * Built a machine learning framework to predict and explain directionality in gene-gene interaction networks using cell-type specific 
  bulk gene expression data
  
* **Software Engineering Intern at Zibrio • June 2021 - August 2021**
  * Ideated and built a dynamic visualization platform with plug-and-play tools using AngularJS and TypeScript for remote patient
monitoring
  * Integrated into proprietary cloud platform and piloted with the 3rd largest hospital system in the US 

* **Stanford Pathology: Bogyo Lab • June 2021 - December 2021**
  * Designed a novel software workflow to transform sequencing data from phage display experiments into
synthesizable peptides for drug discovery, allowing for the identification of highly selective covalent binders
  * Unsupervised hierarchical clustering algorithm saved 3-5 weeks of peptide screening during each phage display campaign

* **Stony Brook Medical School: Duong Lab • April 2020 - December 2020**
  * Created a novel machine learning based risk score to predict neurocognitive disorders and Alzheimer’s Disease
  * Used TensorFlow to identify predictive variables and construct deep neural network with over 90% accuracy (representing a 20% improvement over the state-of-the-art) using non-invasive clinical markers
  * Published in Journal of Alzheimer’s Disease (2021)

* **Baylor College of Medicine: Hodges Lab • March 2019 - December 2020**
  * Constructed a state-of-the-art pipeline to optimize Next Generation Sequencing alignment with Apache
Hadoop/Spark, pipeline is up to 400% faster than conventional aligners with an 85% average alignment accuracy
  * Performed wet-lab experiments to investigate the function of mammalian BAF/PBAF ATPase subunits
  * Presented alignment work at the International Society for Computational Biology ROCKY 2019 Conference 
  
Skills
======
* Programming Languages: Python, C++, C, Java, R, Swift
* Machine Learning: TensorFlow, PyTorch, LightGBM, Scikit-learn, Numpy, Pandas, OpenCV, AWS SageMaker
* Bioinformatics/Cheminformatics: Biopython, PyMol, RDKit, DeepChem, Sequence Aligners
* Software Development Frameworks: React, TypeScript, AngularJS, Apache Spark/Hadoop, PostgreSQL

Teaching
======
* Teaching Assistant for MED275B, Stanford’s premier medical device design class (Spring 2022)
